cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -0,0 +1,14 @@
+[id='odo-release-notes']
+= {odo-title}{nbsp}{odo-ver} Release Notes
```

Comment:
@boczkowska, I haven't seen `{nbsp}` used before. Why doesn't a normal space work here?

Reply:
@jboxman Some products use the {nbsp} to keep the product name and version together (instead of letting them split across lines). It doesn't look like we do in our guidelines, but I think this is fine here.

Reply:
And sentence case here

Reply:
@jboxman what @bergerhoffer said! I can remove it but AFAIK it renders fine so shouldn't be a problem.